README.md

```diff
@@ -1 +1,103 @@
-# covid-workplace-risk
\ No newline at end of file
+# COVID-19 model for Workplace Outbreaks
+> Guo, X.; Gupta, A.; Sampat, A; Wei, C. A Stochastic Contact Network Model for Assessing Outbreak Risk of COVID-19 in Workplaces.
+
+A simulation-based stochastic contact network to assess the cumulative incidence of COVID-19 in workplaces. 
+
+## Getting started
+
+This code is written in `python:3`, and is expected to work in a standard python environment with the following libraries.
+
+### Dependencies
+```
+numpy
```

Comment:
is `logging` a dependency? since it is used in the py file

Reply:
It's available as a standard library in python, like `math`.